crates/rooch-rpc-client/src/client_config.rs

```diff
@@ -15,8 +15,8 @@ use serde_with::serde_as;
 use std::fmt::{Display, Formatter, Write};
 
 pub const DEFAULT_EXPIRATION_SECS: u64 = 30;
-pub const ROOCH_DEV_NET_URL: &str = "https://dev-seed.rooch.network/";
-pub const ROOCH_TEST_NET_URL: &str = "https://test-seed.rooch.network/";
+pub const ROOCH_DEV_NET_URL: &str = "https://dev-seed.rooch.network:443/";
```

Comment:
The default https URL port is 443? There's no need to add `443`.

Reply:
It is related to this issue https://github.com/rooch-network/rooch/issues/869. It is caused by a bug in jsonrpsee and needs to be upgraded. I tried the upgrade and found that it is quite troublesome. The upgrade requires the refactor of RPC result and RPC client. This version does not have enough time and will be dealt with in the next version.